src/screens/UserSettingsScreen/UserSettingsScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { StackNavigationOptions } from '@react-navigation/stack';
+
+import PlainText from '~/components/UI/PlainText';
+import { NavigationData } from '~/navigation/AppNavigator/SettingsNavigator';
+
+import { StyledView } from './UserSettingsScreen.styled';
+
+type Props = NavigationData<'UserSettings'>;
+
+const UserSettingsScreen: React.FC<Props> = () => {
+  return (
+    <StyledView>
+      <PlainText theme="dark">User Settings Screen</PlainText>
+    </StyledView>
+  );
+};
+
+export const UserSettingsScreenNavOptions: StackNavigationOptions = {
+  headerTitle: 'Account Settings',
```

Comment:
I think we will need a translation support for headers. Can you maybe find some solution for it?

Reply:
I have found two solutions, the one with use <Trans /> is more cleaner but it cause type error. You can check and chose one

Reply:
And I found one more alternative. Have a look at my demo branch (`feature/user-info-section-improvement-pt2`).